docs/faq.md

```diff
@@ -49,3 +49,36 @@ intermediates and outputs parallel with the `src` folder and not inside it,
 you can help optimize Git command performance for developers in the repository
 by limiting the number of files Git needs to consider for many common
 operations.
+
+### Why is Scalar a C# tool instead of built directly into Git?
+
+Scalar does a few things that Git currently has no precedent of doing and a few
+things that Git will never do.
+
+Git does not have a concept of running background jobs, which is critical to
+keeping our repositories running smoothly. Git does not have a concept of
+cache-servers, so we need to use the GVFS protocol during `scalar clone` to
+set up the cache-server URL. Scalar also creates the `.scalarCache` directory
+so multiple enlistments share objects from the remote, allowing a second
+`scalar clone` operation to be much faster than the first. All of these concepts
+are on our backlog for contributing to Git, in which case we can transition
+from the Scalar implementation to the Git implementation.
+
+Scalar uses the GVFS protocol for some repositories. This protocol was created
+by Azure Repos to support large repositories before the Git protocol supported
+partial clone and the Git client supported missing objects. In a way, the GVFS
+protocol proved that there was value in creating the partial clone feature.

```

Comment:
What is the intent of this paragraph? Are we trying to say that we have precedent for starting outside of Git but then working with the community to adapt the feature into Git? If so, let's be more explicit about that.